packages/flutter/test/cupertino/route_test.dart

```diff
@@ -847,10 +845,10 @@ void main() {
 
     navigatorKey.currentState.push(route2);
     await tester.pumpAndSettle();
-    verify(navigatorObserver.didPush(any, any)).called(greaterThanOrEqualTo(1));
+    expect(navigatorObserver.invocations, contains(NavigatorInvocation.didPush));
 
     await tester.dragFrom(const Offset(5, 100), const Offset(100, 0));
-    verify(navigatorObserver.didStartUserGesture(any, any)).called(1);
+    expect(navigatorObserver.invocations, contains(NavigatorInvocation.didStartUserGesture));
```

Comment:
The old test was a little more strict (`called(1)`)?

Reply:
Took another pass